spec/models/politician_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Politician, type: :model do
+  let(:politician) { build(:politician) }
+  
+  # validate_presence_of to all items
+  [:name, :political_party, :age, :type, :state, :image_url].each do |field|
+    it { should validate_presence_of(field) }
```

Comment:
usa o is_expected.to ao invés de should, o rubocop não gosta mais do should #chateadu